<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MFifoStatsUi</class>
 <widget class="QWidget" name="MFifoStatsUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MFifo Stats</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Main_label_3">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>MFifo Stats Console</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_zoomOut">
         <property name="text">
          <string>ZO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_zoomIn">
         <property name="text">
          <string>ZI</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_zoomFit">
         <property name="text">
          <string>ZF</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
